views/partials/searchCard.handlebars

```diff
@@ -0,0 +1,25 @@
+<div class="search-card">
+  <div class="search-left">
+    <div class="search-dropoff-info">
+      <h1 class="search-name">{{title}}</h1>
+      <h2 class="search-address">{{address}}</h2>
+      <h2 class="search-phone">{{phone}}</h2>
+    </div>
+    <div class="search-dropoff-options">
+      <p>Drop off 
+        {{#if isSupportDropOff}} <span class="valid-symbol">✓</span> {{else}} <span class="invalid-symbol">✗</span> {{/if}}
```

Comment:
You could factor this into an inline partial

https://handlebarsjs.com/guide/partials.html#inline-partials

Reply:
Would the benefit of this be that I can make an inline partial for the if statement, then use that partial for isSupportDropOff and isSupportMailIn instead of having to re-type the if statement?